reference/library/src/webapp/js/headscripts.js

```diff
@@ -817,15 +816,6 @@ function rewriteWebLearnHref(link) {
     }
 }
 
-// Sets the target property to '_blank' so that insecure content is opened in a new window
-function addTargetBlank(link) {
-    if(link.href && link.href.match(/^http:/)) {
```

Comment:
I don't understand what this is fixing? While this will cause http://weblearn.ox.ac.uk links to be re-written, don't we still want that and isn't it the case that that isn't the major issue here?